tests/greenIT/input/url.yaml

```diff
@@ -0,0 +1,6 @@
+# Analyse l'URL collectif.greenit.fr 
+- name : 'HyperGlosae'
+  url : 'http://localhost:3000/'
```

Comment:
@vallhallalm Notez que la page d'accueil est la seule page qui n'a pas encore réellement fait l'objet d'un design et encore moins d'un "eco-design". Je comprends l'intérêt d'analyser sa qualité pour illustrer une amélioration, mais par contre ce serait un peu trompeur d'analyser uniquement celle-là.

Reply:
@benel Effectivement je tacherai de complexifier le scenario de green IT analysis quand j'aurais réussi à faire passer le test sur la pull request